Description: Install kcmgtk4.desktop in SERVICES and not in XDG_APPS
Author: Fathi Boudra <fabo@debian.org>
Last-Update: 2008-05-01

--- a/kcm_gtk/CMakeLists.txt
+++ b/kcm_gtk/CMakeLists.txt
@@ -28,7 +28,7 @@ INSTALL(
 
 INSTALL(
 	FILES kcmgtk4.desktop
-	DESTINATION ${XDG_APPS_INSTALL_DIR}
+	DESTINATION ${SERVICES_INSTALL_DIR}
 )
 
 INSTALL(
